docsite/index.css

```diff
@@ -169,6 +169,10 @@ footer {
   }
 }
 
+footer dl > div {
+  place-self: start center;
+}
+
```

Comment:
can you change it [here instead](https://github.com/argyleink/open-props/blob/main/docsite/index.css#L159) please =)

Reply:
I had a guess about this too. I'll change it now.

Reply:
done